cardano-db-sync/src/ApiTest.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module ApiTest
+where
+import           Data.Proxy
+import           Data.Void (Void)
+import qualified Data.ByteString.Lazy as BSL
+import           Ouroboros.Consensus.Config (TopLevelConfig, configBlock)
+import           Ouroboros.Consensus.Network.NodeToClient (clientCodecs, ClientCodecs)
+import           Ouroboros.Consensus.Node.NetworkProtocolVersion (
+                    nodeToClientProtocolVersion , supportedNodeToClientVersions)
+import           Ouroboros.Consensus.Node.Run (RunNode, nodeNetworkMagic)
+import           Ouroboros.Network.Mux (AppType (..), OuroborosApplication)
+import           Ouroboros.Network.NodeToClient (ClientSubscriptionParams (..),
+                    ConnectionId, LocalAddress,
+                    NodeToClientProtocols (..),
+                    NetworkClientSubcriptionTracers,
+                    NodeToClientVersionData (..),
+                    ncSubscriptionWorker,
+                    newNetworkMutableState,
+                    versionedNodeToClientProtocols)
+
+import           Ouroboros.Consensus.Node.NetworkProtocolVersion (NodeToClientVersion)
+import           Ouroboros.Network.Protocol.Handshake.Version (DictVersion, Versions, foldMapVersions)
+import qualified Ouroboros.Network.Snocket as Snocket
+
+import qualified Ouroboros.Network.NodeToClient (NodeToClientVersion)
+import           Control.Monad.Class.MonadST (MonadST)
+
+import           Prelude
+
+subscribe ::
+  ( RunNode blk , MonadST m1 )
+  => Snocket.LocalSnocket
+  -> TopLevelConfig blk
+  -> NetworkClientSubcriptionTracers
+  -> ClientSubscriptionParams ()
```

Comment:
This is not related, but it seems that `a` in `ClientSubscriptionParams a` is not used.